src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet">
+  </head>
+
+  <body class="body">
+    <section class="product" data-qa="card">
+      <div class="product__photo">
+        <img
+          class="product__photo_img"
+          src="images/imac.jpeg"
+          alt="monoblock"
+        >
+      </div>
+      <h3 class="product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+      <p class="product__code">Код товара: 195434</p>
+      <div class="product__rate">
+        <div class="product__rate_stars stars__4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="product__rate_amount">Отзывов: 5</span>
```

Comment:
Don't use snake_case in element names. Use kebab-case instead.
```suggestion
        <span class="product__rate-amount">Отзывов: 5</span>
```